lib/livebook_web/live/hub/secret_form_component.ex

```diff
@@ -91,10 +100,7 @@ defmodule LivebookWeb.Hub.SecretFormComponent do
         {:noreply, assign(socket, changeset: Map.replace!(changeset, :action, :validate))}
 
       {:transport_error, error} ->
-        {:noreply,
-         socket
-         |> put_flash(:error, error)
```

Comment:
Shouldn't this have worked then? Why it didn't? 🤔

Reply:
Ah... it worked but it was hidden in the z-index. Got it.

Reply:
It did, but it's behind the modal overlay